src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -404,6 +423,68 @@
 # Create SES Service For edxapp Emails #
 ########################################
 
+edxapp_ses_domain_identity = ses.DomainIdentity(
+    "edxapp-ses-domain-identity", domain=edxapp_config.require("mail_domain")
+)
+edxapp_ses_verification_record = route53.Record(
+    "edxapp-ses-domain-identity-verification-dns-record",
+    zone_id=edxapp_zone_id,
+    name=edxapp_ses_domain_identity.id.apply("_amazonses.{}".format),
+    type="TXT",
+    ttl=FIVE_MINUTES,
+    records=[edxapp_ses_domain_identity.verification_token],
+)
+edxapp_ses_domain_identity_verification = ses.DomainIdentityVerification(
+    "edxapp-ses-domain-identity-verification-resource",
+    domain=edxapp_ses_domain_identity.id,
+    opts=ResourceOptions(depends_on=[edxapp_ses_verification_record]),
+)
+edxapp_mail_from = ses.MailFrom(
+    "edxapp-ses-mail-from-domain",
+    domain=edxapp_ses_domain_identity_verification.domain,
+    mail_from_domain=edxapp_ses_domain_identity_verification.domain.apply(
+        "bounce.{}".format
+    ),
+)
+# Example Route53 MX record
+edxapp_ses_domain_mail_from_mx = route53.Record(
+    f"edxapp-ses-mail-from-mx-record-for-{env_name}",
+    zone_id=edxapp_zone_id,
+    name=edxapp_mail_from.mail_from_domain,
+    type="MX",
+    ttl=FIVE_MINUTES,
+    records=["10 feedback-smtp.us-east-1.amazonses.com"],
+)
+ses_domain_mail_from_txt = route53.Record(
+    "edxapp-ses-domain-mail-from-text-record",
+    zone_id=edxapp_zone_id,
+    name=edxapp_mail_from.mail_from_domain,
+    type="TXT",
+    ttl=FIVE_MINUTES,
+    records=["v=spf1 include:amazonses.com -all"],
+)
+edxapp_ses_domain_dkim = ses.DomainDkim(
+    "edxapp-ses-domain-dkim", domain=edxapp_ses_domain_identity.domain
+)
+edxapp_ses_dkim_records = []
+for loop_counter in range(0, 3):
+    edxapp_ses_dkim_records.append(
```

Comment:
Not sure I follow what we're trying to do appending dkim records

Reply:
That's just adding it to the list object. Right now we're not doing anything with it so I can remove it, but if we want to pass it along to anything else or export it then that's where we would use that list.

Reply:
Mostly copy :spaghetti: from an example

Reply:
I would either remove or maybe add a comment explaining this just for record keeping